toml/src/main/kotlin/org/rust/toml/crates/local/TestCrateResolverService.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.toml.crates.local
+
+import org.jetbrains.annotations.TestOnly
+
+@TestOnly
```

Comment:
```suggestion
```
This class is registered in XML so it should not be excluded. Otherwise, the compilation of other plugins depending on IJ Rust would be broken. See `TestCargoProjectsServiceImpl` for example